.github/workflows/pr-builder.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           set -e
           export LOG_LEVEL=debug
           make build_with_coverage_only OS=$(go env GOOS) ARCH=$(go env GOARCH)
```

Comment:
Shall we integrate the frontend build to this `build_with_coverage_only ` and then just upload the coverage?

@ThaminduDilshan Do we need a `build_with_backend_coverage_only` as well?
If so we can have three:

- `build_with_coverage_only`
- `build_with_backend_coverage_only`
- `build_with_frontend_coverage_only`